README.md

```diff
@@ -49,54 +49,83 @@ Another option is to open it with Control-click: it'll immediately register the
 ### features
 
 - [ ] images from tags
+
 - [ ] indicates when track is in playlist
+
 - [ ] configure replay gain from settings
+
 - [ ] display tracks/albums/artists count in settings
+
 - [ ] allow reseting database from settings
+
 - [ ] smaller screen support (UI refactor)
 
 ### tools
 
 - [ ] App automated end to end tests
+
 - [ ] more technical documentation (install & release process notably)
 
 ### release
 
 - [ ] github page
+
 - [ ] usage statistics
+
 - [ ] references
+
   - [ ] Electron's [app list](https://www.electronjs.org/apps)
+
   - [ ] Svelte's [showcase](https://svelte-community.netlify.app/showcase)
+
   - [ ] [Snap store](https://snapcraft.io/)
 
 ### Bugs and known issues
 
-1. Playlist are not updated on tracks removal/changes
+1. Playlist models are not updated on tracks removal
+
 1. Undetected live changes: remove tracks and re-add them. This is a linux-only issue with chokidar
-   - https://github.com/paulmillr/chokidar/issues/917
-   - https://github.com/paulmillr/chokidar/issues/591
-1. Files renamed or moved to other watched folders are removed and re-added. This is a limitation with chokidar
-   - https://github.com/paulmillr/chokidar/issues/303
+
+   - [Issue #917](https://github.com/paulmillr/chokidar/issues/917)
+
+   - [Issue #591](https://github.com/paulmillr/chokidar/issues/591)
+
 1. When loading new folders, enqueuing or going to album details will give incomplete results. Going back and forth won't load new data
+
 1. Security: clean html in artist/album names (wrapWithRefs returns injectable markup)
-1. AppImage, when used with AppImageLauncher, fail to auto update: https://github.com/electron-userland/electron-builder/issues/4046#issuecomment-670367840
+
+1. AppImage, when used with AppImageLauncher, [fail to auto update](https://github.com/electron-userland/electron-builder/issues/4046#issuecomment-670367840)
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `5`, was `1`](https://app.codacy.com/gh/feugy/melodie/pullRequest?prid=6368165)